hc/__init__.py

```diff
@@ -0,0 +1,2 @@
+import pymysql
+pymysql.install_as_MySQLdb()
```

Comment:
I would prefer to have this in `settings.py` and execute it conditionally if `os.getenv("DB") == "mysql"`

Reply:
Yes it would be more specific and accurate.